content/developers/dogstatsd/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: DogStatsD
+kind: documentation
+description: Introduction and setup instructions for DogStatsD.
+aliases:
+  - /guides/dogstatsd/
+  - /guides/DogStatsD/
+further_reading:
+- link: "developers/metrics"
+  tag: "Documentation"
+  text: Learn more about Metrics
+- link: "developers/libraries"
+  tag: "Documentation"
+  text: Official and Community-contributed API and DogStatsD client libraries
+- link: "https://github.com/DataDog/dd-agent/blob/master/dogstatsd.py"
```

Comment:
This points to the Agent 5 repo, shouldn't we point to the agent 6 repo instead?